pkg/universe.dagger.io/examples/todoapp/plan.cue

```diff
@@ -0,0 +1,49 @@
+package todoapp
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/netlify"
+	"universe.dagger.io/git"
+
+	"universe.dagger.io/examples/todoapp/pkg/yarn"
+)
+
+dagger.#Plan & {
+	client: {
+		filesystem: {
+			"./_build": write: contents: actions.build.output
+			"./app/": write: contents:   actions.develop.output
+			"./app": read: {
+				contents: dagger.#FS
+				exclude: [
+					".git",
+					"node_modules",
+					"*.md",
+				]
+			}
+		}
+		env: {
+			NETLIFY_TOKEN: dagger.#Secret
+			APP_NAME:      string
+		}
+	}
+	actions: {
+		// Checkout a fresh copy of the todoapp repo
+		develop: git.#Pull & {
+			remote: "https://github.com/mdn/todo-react"
+			ref:    "master"
+		}
```

Comment:
I know that this cleans up things in the the `todoapp` dir, but is it realistic? The starting point was _I have a yarn app and I want to use Dagger for CI/CD & other automation_. This change makes it _I start with Dagger which creates everything else, including a clone of my app_. I think that this new starting point will make it a lot more difficult for users to see how to get started. I am always wary of tools that can solve it all, and this is a step in that direction.

Reply:
Agreed, it's not realistic. I will try another approach.